src/hooks/useCosmostationWallet.ts.bak

```diff
@@ -0,0 +1,209 @@
+import { useCallback, useEffect, useState } from "react";
```

Comment:
could you let me know what this file is for?

Reply:
added this file accidentally